src/parser.yy

```diff
@@ -67,6 +70,8 @@
             (Cur).filenames     = YYRHSLOC(Rhs, 0).filenames;   \
         }                                                       \
     } while (0)
+
+    using std::move;
```

Comment:
I know std::move is now *everywhere*, but I think the extra 5 characters each time do make it clearer when scanning through the code. This does also get included in a header file, although currently that only contaminates main.cpp.

Reply:
It makes some lines longer than 110 chars, forcing line wraps. I'll change it, but the inclusion in header was unintended.

Reply:
Most of these are due to not having access to `api.cpp.automove`, which requires version 3.2. What platforms are holding us back? Have any updated recently?

Reply:
Apparently Ubuntu 18 is stuck on 3.0. That's at least one upper bound.

Reply:
The `using std::move` is no more. It was replaced by the `Mov` abomination, which incidentally makes it almost ready as-is for `api.cpp.automove` whenever we move to Bison 3.2.